docs/usage-examples.txt

```diff
@@ -0,0 +1,67 @@
+.. _laravel-usage-examples:
+
+==============
+Usage Examples
+==============
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: set up, runnable, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+Usage examples show runnable code examples to demonstrate frequently used MongoDB
+operations. Each usage example includes the following components:
+
+- Explanation of the MongoDB operation
+- Example controller code that runs the MongoDB operation
+- Example view code that renders the result
+- Output displayed by the view
+
+How to Run the Usage Examples
+-----------------------------
+
+The usage examples are designed to run operations on a MongoDB deployment that contains
+the MongoDB Atlas sample datasets. When you run the example code without this sample data,
+the output might not match. 
+
+You can run the usage examples from the Laravel web application and MongoDB Atlas cluster that you
+set up in the :ref:`laravel-quick-start` guide. After completing the Quick Start, ensure that your
+application meets the following requirements:
+
+- Connects to the ``movies`` collection in the Atlas sample datasets
+- Contains the ``MovieController.php`` and ``browse_movies.blade.php`` files
+
+Then, follow this tutorial to add the usage example code to your Laravel application and view 
+the expected results.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Add the Controller Code to Your Application
+      
+      Copy the example code from the :guilabel:`Controller File Code` tab on the usage example
+      page and paste it into the ``MovieController.php`` file.
+
+   .. step:: Add the View Code to Your Application
+      
+      Copy the example code from the :guilabel:`View File Code` tab on the usage example page and
+      paste it into the ``browse_movies.blade.php`` file.
+
+   .. step:: Run the Usage Example Code
+
+      To run the example controller code and view the results, follow the instructions on the
+      usage example page.
```

Comment:
Suggestion:

I think presenting this information might not make sense before a reader gets the chance to see a Usage Example page. As mentioned in a [previous comment](https://github.com/mongodb/laravel-mongodb/pull/2767#discussion_r1523347083), it should delegate some of the explanation to the specific usage examples since they may differ depending on the operation.

Start with information on when this applies, e.g.
"
Each Usage Example page features example code in the ``Controller File Code`` and ``View File Code`` tabs that you can copy and paste into your Laravel application.

Unless otherwise specified, you can add Usage Example sample code to your application by performing the following actions:
- Add the code from the ``Controller File Code`` tab to your ``MovieController.php`` file in the ... directory
- Add the code from the ``View File Code`` tab to your ``browse_movies.blade.php`` file in the ... directory.

Follow the instructions on the Usage Example page to run the code and view the output.
"

Reply:
Ah okay, so including tutorial steps doesn't seem necessary; changed